SwiftUI/Model/SearchOperation/SearchOperation.mm

```diff
@@ -84,6 +84,9 @@ - (void)addIndexSearchResults:(std::vector<zim::Archive>)archives {
         NSUUID *zimFileID = [[NSUUID alloc] initWithUUIDBytes:(unsigned char *)result.getZimId().data];
         NSString *path = [NSString stringWithCString:item.getPath().c_str() encoding:NSUTF8StringEncoding];
         NSString *title = [NSString stringWithCString:item.getTitle().c_str() encoding:NSUTF8StringEncoding];
```

Comment:
I haven't experience the title being empty here, but we can do a fallback as suggested.